startup/routes.js

```diff
@@ -1,8 +1,26 @@
 const express = require('express')
+const rateLimit = require('express-rate-limit')
+
+const ApiError = require('../helpers/apiError')
+const globalErrorHandler = require('../middleware/globalErrorHandler')
+
 const example = require('../routes/example')
 
 module.exports = function (app) {
   app.use(express.json())
 
-  app.use('/api/users', example)
+  const limit = rateLimit({
```

Comment:
Place it above module.exports block and pass only basic routes inside. It should work fine